tests/test_tools.py

```diff
@@ -353,11 +353,11 @@ def test_create_authorid_profiles(self, client):
     def test_subdomains(self):
         # ensure that two part top-level domains are handled appropriately
         # e.g. "edu.cn", "ac.uk"
-        assert openreview.tools.subdomains('michael@mails.tsinghua.edu.cn') == ['mails.tsinghua.edu.cn', 'tsinghua.edu']
```

Comment:
why did you change the assertions here?

Reply:
Because the function `openreview.tools.subdomains` was doing two things" splitting the emails and then getting the subdomains for the domains of the emails. I extracted that logic and split the emails before calling the `subdomains` function